packages/medusa/src/services/order-edit.ts

```diff
@@ -382,4 +383,27 @@ export default class OrderEditService extends TransactionBaseService {
 
     return orderEdit
   }
+
+  async requestConfirmation(
+    orderEditId: string,
+    userId: string
```

Comment:
suggestion: to keep the consistency `context: { loggedInUserId }` as we will pass other arguments in the context in a close future and it would be consistent with the other methods